src/Frontend/Page/Overview.hs

```diff
@@ -101,6 +103,8 @@ instance ToHtml PageOverviewOfWildIdeas where
             div_ [class_ "ideas-list"] $ toHtml ideasAndNumVoters
 
 instance Page PageOverviewOfWildIdeas where
+    -- Any logged in user is authorized since findWildIdeasBySpace is already selecting the right view
+    isAuthorized = userPage
```

Comment:
does that mean that we can visit enemy idea spaces, only they will appear empty?  will that be fixed by the ideaspace caps change?